app/src/org/commcare/views/widgets/DatePrototypeFactory.java

```diff
@@ -0,0 +1,19 @@
+package org.commcare.views.widgets;
+
+import android.content.Context;
+
+import org.javarosa.form.api.FormEntryPrompt;
+
+public class DatePrototypeFactory {
```

Comment:
It's still useful while we're making the cancel button optional via appearance attribute